src/hub.js

```diff
@@ -230,35 +231,118 @@ const onReady = async () => {
 
   const enterScene = async (mediaStream, enterInVR, hubId) => {
     const scene = document.querySelector("a-scene");
+    scene.style.cursor = "none";
     scene.renderer.sortObjects = true;
     const playerRig = document.querySelector("#player-rig");
     document.querySelector("canvas").classList.remove("blurred");
     scene.render();
 
     if (enterInVR) {
       scene.enterVR();
+      if (isMobile) {
```

Comment:
This block can move into a component on the scene and then you should be able to remove all the `componentinitialized` checks. Combine that with converting the `register` funcitons into constructor arguments and a lot of this code goes away.